### ISSUE TITLE
Hyper-Parameters

### ISSUE THREAD

**Issue by huichen5796** [huichen5796/hiwi-2023#2]
*ResNet50*
- SGD
- batch size ***?*** 512 / 32
- initial learning rate 0.003
- momentum 0.9
- weight decay 2e-5
- warm up epochs 10
- plateau scheduler
    - patience: 10 epochs
    - decay rate: 0.1
- stop: lr below 1e-6

*BiT-HyperRule*
- SGD
- initial learning rate of 0.003 
- momentum 0.9
- batch size ***?*** 512 / 32
- learning rate should be decayed by a factor of 10 at 30%, 60% and 90% of the training steps

*ViT-B*
- SGD 
- momentum 0.9
- learning rate 0.003 
- batch size 512
- 500 steps with cosine decay learning rate
- 3 warm-up epochs
- gradient clipping at norm 1

*DeiT-S*
- patch size 16
- Adam optimizer
- learning rate 1e-4 
- momentum 0.9
- weight decay 1e-5
- plateau scheduler with minimal learning rate 1e-6 
- warm up for 10 epochs 
- batch size 64
